scripts/pdg.sh

```diff
@@ -77,6 +78,32 @@ main() {
             --output "${pdg}" \
         > "${test_dir}/pdg.log"
     )
+    (
+        export RUST_BACKTRACE=full # print sources w/ color-eyre
+        export RUST_LOG=error
+        export PDG_FILE="${pdg}"
+        local deps_dir="${CARGO_TARGET_DIR:-target}/${profile_dir_name}/deps"
+        local libc_dep="$(ls -1 --sort=time -r ${deps_dir}/liblibc-*.rlib | head -n1)"
+        cargo run \
+            --bin c2rust-analyze \
+            "${profile_args[@]}" \
+            -- \
+            --crate-name "${crate_name}" \
+            --edition 2021 \
+            --crate-type bin \
+            --sysroot $(rustc --print sysroot) \
+            "-C" \
+            "opt-level=3" \
+            "-C" \
+            "embed-bitcode=no" \
+            "--cfg" \
+            "feature=\"c2rust-analysis-rt\"" \
+            "-C" \
+            "metadata=4095517b1921578c" \
+            "-L" "dependency=${deps_dir}" \
+            "--extern" "libc=${libc_dep}" \
```

Comment:
We also need `c2rust_bitfields`.

See https://github.com/immunant/lighttpd-rust/blob/385ca548101242cfb8f8bc61533f1073042b5036/lighttpd_rust_amalgamated/c2rust-analyze.sh#L36-L37.